src/main/java/uk/gov/pay/ledger/filters/LoggingMDCRequestFilter.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.pay.ledger.filters;
+
+import org.slf4j.MDC;
+
+import javax.ws.rs.container.ContainerRequestContext;
+import javax.ws.rs.container.ContainerRequestFilter;
+import java.io.IOException;
+import java.util.Optional;
+
+import static uk.gov.pay.logging.LoggingKeys.GATEWAY_ACCOUNT_ID;
+import static uk.gov.pay.logging.LoggingKeys.LEDGER_EVENT_ID;
+
+public class LoggingMDCRequestFilter implements ContainerRequestFilter {
+
+    public static final String PARENT_EXTERNAL_ID = "parent_external_id";
```

Comment:
might be good to make this key "parent_transaction_external_id" so it's more obvious what it refers too

Reply:
Done.

Reply:
Sorry I meant the other one, we should have `TRANSACTION_EXTERNAL_ID` and `PARENT_TRANSACTION_EXTERNAL_ID` I think